### ISSUE TITLE
F# "//#region" not working

### ISSUE THREAD

**Issue by vsfeedback** [dotnet/fsharp#14925]
_This issue has been moved from [a ticket on Developer Community](https://developercommunity.visualstudio.com/t/F-region-not-working/10312826)._

---
[severity:It's more difficult to complete my work]
Coming from C#, "#region" help A LOT for lessening cognitive overload. I fear that F# in VStudio don't allow such a feature. I hope I'm wrong, otherwise VSCode understood why for many F# user this a MUST feature for code organisation, at least for the prototype/scripting phase where many block of code might temporarily coexist

---
#### Feedback Bot on 3/17/2023, 02:10 AM: 

(private comment, text removed)

---
### Original Solutions
(no solutions)

**Comment by Xyncgas** [comment]
Meanwhile you can use an auto open module with random names such as ``aoiewfjaoiejfaoiejfaoife aoiewfaief -a908341``

**Comment by abelbraaksma** [comment]
Yes, I think adding `#region` would be beneficial. It will need a language suggestion, though.

See this comment as well: https://github.com/fsharp/fslang-suggestions/issues/278#issuecomment-386617042
